components/Map.tsx

```diff
@@ -1,29 +1,29 @@
 import * as React from "react";
-import { useState, useEffect, useRef, useCallback } from "react";
```

Comment:
VSCode linting did this automatically. Let me know if you'd prefer me to revert this.